### PR TITLE
Include button for launching jupyterlab layout in repr

### PR DESCRIPTION
**Pull request by ian-r-rose**
This is an experimental UI element to help ease dashboard setup in a JupyterLab environment. Doesn't make sense to merge without a paired release of `dask-labextension`, but opening for discussion/visibility.

dask-layout-2

Edit: this is now ready for review, and works with `dask-labextension==6.0.0`.

### CODE REVIEW

**Review by gjoseph92 — COMMENTED** [review]
Looking forward to this!

File: distributed/client.py [hunk 1]
```diff
@@ -1145,6 +1145,14 @@ def __repr__(self):
             return f"<{self.__class__.__name__}: No scheduler connected>"
 
     def _repr_html_(self):
+        try:
+            import dask_labextension  # noqa: F401
+
+            # TODO: also guard with version check
```
[the remark this answers is not in the document]
Comment (ian-r-rose) [reply distributed/client.py]: Nice idea

**Comment by mrocklin** [comment]
It looks like dask-labextension 6.0.0 is out.  The CI failures are unrelated.  Merging this in.  Thank you @ian-r-rose

**Comment by mrocklin** [comment]
Question though: I often run JupyterLab in my base environment and then run Dask in a separate conda environment.  In this case our code won't have dask-labextension in the same space. Correct?  I think that this solution only works if I'm running Dask in the same environment where JLab is running.  Correct?

**Pull request merged**